charts/nexus3/templates/statefulset.yaml

```
@@ -88,16 +85,10 @@ spec:
         - name: download-plugins
           image: {{ include "nexus3.image" .Values.bashImage }}
           imagePullPolicy: {{ .Values.bashImage.pullPolicy }}
+          {{- with .Values.nonRootSecurityContext }}
```

```suggestion
          {{- with .Values.initNonRootSecurityContext }}
```

I think this is a more descriptive name (also not a pod security context).